packaging/standalone/src/main/distribution/shell-scripts/bin/utils

```diff
@@ -234,34 +234,22 @@ stopit() {
   else
     printf "Stopping $FRIENDLY_NAME [$NEO4J_PID]..."
     x=0
-    while [ $x -lt $TIMEOUT ] && [ "$NEO4J_PID" != "" ]  ; do
+    while [ "$NEO4J_PID" != "" ]  ; do
       kill $NEO4J_PID 2>/dev/null
       printf "."
       sleep 1
       checkstatus
+      if [ $x -ge $TIMEOUT ] ;then
+	  echo ""
+	  echo "$FRIENDLY_NAME [$NEO4J_PID] is taking more than ${TIMEOUT}s to stop"
+	  echo "Stopping $FRIENDLY_NAME [$NEO4J_PID]..."
```

Comment:
This looks strange. We're printing `Stopping neo4j...` _after_ saying `...neo4j is taking more than bla to stop`.